test/resources/validate/good-validation/environment.yaml

```diff
@@ -10,3 +10,24 @@ image:
   publish:
     location: mcr
     visibility: public
+environment:
+  metadata:
+    Cuda:
+      CudaVersion: "11.1.1"
+      CuDnnVersion: "8.0.5.39"
+      NcclVersion: "2.8.4"
+    MachineLearningFramework:
+      name: TensorFlow
+      version: "2.8"
+    Mpi:
+      name: OpenMpi
+      version: "4.1.0"
+    Purpose:
+      HasTrainingSupport: true
+      HasInferencingSupport: false
+    Runtime:
+      name: Python
+      version: "3.8"
+    Os:
+      name: Ubuntu
+      version: "20.04"

```

Comment:
Please don't add this back in. It's not supported.

Reply:
my bad, did not see that I committed this